aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -216,6 +217,11 @@ fn is_approved_gov_script(
     }
 }
 
+struct AptosVMRandomnessConfig {
```

Comment:
fixed, but something still has to stay inside crate `aptos-vm` because fetch requires a `resolver: &impl AptosMoveResolver` which is only available in this crate.

Reply:
Btw, you can use `&impl ConfigStorage` trait and move it out (not too important I guess). The only reason why resolver is used is because we have one config which is inside resource group